primeqa/services/grpc_server/protos/indexer.proto

```diff
@@ -13,7 +13,7 @@ service Indexer {
     rpc GetIndexes(GetIndexesRequest) returns (GetIndexesResponse);
 }
 
-message IndexerComponent {
+message Indexer {
```

Comment:
Changes to the proto needs to be in sync with the proto files in the primeqa-orchestrator

Reply:
to be raised as a separate PR in primeqa-orchestrator cc: @kpfadnis